backend/src/appointment/controller/auth.py

```diff
@@ -24,7 +24,8 @@ def logout(
 ):
     """Sets a minimum valid issued at time (time). This prevents access tokens issued earlier from working."""
     if deny_previous_tokens:
-        subscriber.minimum_valid_iat_time = datetime.datetime.now(datetime.UTC)
+        # Reduce the parry-window for webhooks. This will hopefully prevent the login immediately logged out issue.
+        subscriber.minimum_valid_iat_time = datetime.datetime.now(datetime.UTC) - datetime.timedelta(minutes=1)
```

Comment:
Good workaround! Should we mark this as FIXME or TODO and recheck, when we switched Auth?